am-lib/src/main/java/uk/gov/hmcts/reform/amlib/internal/repositories/AccessManagementRepository.java

```diff
@@ -53,4 +55,8 @@
 
     @SqlQuery("select access_management_type from roles where role_name = :roleName")
     AccessType getRoleAccessType(String roleName);
+
+    @SqlQuery("select service_name, resource_type, resource_name from default_permissions_for_roles where role_name = :roleName and default_permissions_for_roles.permissions = 1")
```

Comment:
I haven't read the story but current implementation works only when CREATE is solely permission. Of course user can create resource when CREATE permission is mixed with other permissions .e.g. resource definition with permission equal '3' that means 'CREATE` and 'READ' should also be included in outcome.

```suggestion
    @SqlQuery("select service_name, resource_type, resource_name from default_permissions_for_roles where role_name = :roleName and permissions & 1 = 1")
```